modules/mri_violations/test/TestPlan.md

```diff
@@ -50,6 +50,12 @@
    that the help content about MRI Protocol Check Violation is showing up
    and is up-to-date.
 
+### Dashboard Widget - "My Tasks" for Violated scans
+1. Ensure the total of Violated scans corresponds with the correct 
```

Comment:
Are there site or permission related things that need to be verified while testing?

Reply:
Hi @driusan, I added a line about what permissions are needed from the mri_violations module to view the widget.

Reply:
Does the text of the label change depending on the permission? Or the filtering automatically done when clicking on the link?

Reply:
@driusan the Violated scans row in My Tasks will not display if the user doesn't have either of the permissions.

Reply:
Yeah, but in most of them the label changes from "Sites: all" to "Sites: user sites".. is that not the case for this one?

Reply:
@driusan The label is always 'Violated scan(s)' for this widget.

Reply:
@driusan see @nicolasbrossard comment.